css/homepage.scss

```diff
@@ -195,3 +195,6 @@ p.buttonContainer {
     }
   }
 }
+div.hidden {
```

Comment:
[Préférer](http://stackoverflow.com/questions/20518143/hiding-an-element-difference-between-javascript-attribute-and-css-style) l'attribut [`hidden` ](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden).